app/javascript/stylesheets/general.scss

```diff
@@ -0,0 +1,711 @@
+// scss
+@import "../../../node_modules/mappy-breakpoints/mappy-breakpoints.scss";
+
+$breakpoints: (
+  "small": 375px,
+  "large": 960px,
+);
+
+//////////////////////////////////////////////////////////////////////////
+//COLORS
+$primary-color: #6600cc;
+$dark-color: #252636;
+$white-color: #ffffff;
+$gray-color: #aaaaaa;
+$shadow-color: #f5f4f9;
+
+//////////////////////////////////////////////////////////////////////////
+//HOMEPAGE
+.hero-img {
+  background-image: linear-gradient(
+      rgba(37, 38, 54, 0.2),
+      rgba(37, 38, 54, 0.2)
+    ),
+    url("./../images/banner.svg");
+  height: 45rem;
+  background-position: left;
+  background-repeat: no-repeat;
+  background-size: cover;
+  position: relative;
+  @include mappy-bp(small large) {
+    height: 34.375rem;
+  }
+
+  .hero-title {
+    color: $white-color;
+    font-style: normal;
+    font-weight: 800;
+    font-size: 4rem;
+    line-height: 5rem;
+    display: table-caption;
+    margin-left: 1rem;
+    margin-top: 19rem;
+  }
+}
+
+.home-title {
+  color: $dark-color;
+  font-weight: 600;
+  font-size: 2rem;
+
+  @include mappy-bp(small large) {
+    font-size: 1.313rem;
+    margin-left: 1rem;
+  }
+}
+
+.new-classes {
+  margin-left: 9rem;
+  padding-top: 7rem;
+
+  @include mappy-bp(small large) {
+    margin-left: 1rem;
+    padding-top: 5rem;
+  }
+}
+.filter-wrapper {
+  width: 100%;
+  display: flex;
+  overflow-x: auto;
+  margin: 0.75rem 0;
+
+  &::-webkit-scrollbar {
+    width: 0;
+  }
+}
+
+.new-classes-wrapper {
+  width: 100%;
+  display: flex;
+  overflow-x: auto;
+
+  &::-webkit-scrollbar {
+    width: 0;
+  }
+}
+
+.categories {
+  margin-left: 9rem;
+  padding-top: 7rem;
+
+  @include mappy-bp(small large) {
+    margin-left: 1rem;
+    padding-top: 5rem;
+  }
+
+  .categories-wrapper {
+    display: grid;
+    grid-gap: 1rem;
+
+    @include mappy-bp(small large) {
+      grid-template-columns: repeat(2, 10.25rem);
+      margin-top: 1rem;
+    }
+  }
+}
+
+.popular {
+  margin-left: 9rem;
+  padding-top: 7rem;
+
+  @include mappy-bp(small large) {
+    margin-left: 0;
+    padding-top: 5rem;
+  }
+
+  .popular-wrapper {
+    @include mappy-bp(small large) {
+      margin-left: 1rem;
+    }
+  }
+}
+
+//////////////////////////////////////////////////////////////////////////
+//NAVBAR
```

Comment:
**issue (blocking)**: when I suggested a single entry point file for styles I did not mean to put all the styles in the same file. You can have separate files like you had before, but then `@import` all the files here instead of individually in the react components. That way, when SCSS compiles there is a single file, but while developing having separate files is much better.

Reply:
aaah ok, I was a little confused about the idea, no I get it